frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -421,409 +423,378 @@ class BitBox02 extends Component<Props, State> {
         if (!showWizard) {
             return <Settings deviceID={deviceID}/>;
         }
+
         const readDisclaimers = agreement1 && agreement2 && agreement3 && agreement4 && agreement5;
-        // TODO: move to wizard.tsx
         return (
-            <div className="contentWithGuide">
-                {
-                    waitDialog && (
-                      <WaitDialog title={waitDialog.title}>
-                          {waitDialog.text ? waitDialog.text : t('bitbox02Interact.followInstructions')}
-                      </WaitDialog>
-                    )
-                }
-                <div className="container">
-                    <Status hidden={attestationResult !== false}>
-                        {t('bitbox02Wizard.attestationFailed')}
-                    </Status>
```

Comment:
Before we had the attestation warning on all setup steps, moved to inline to pairing and unlock view. Could add to other steps if needed, but it cannot overlay at the moment.